src/main/java/org/opensearch/ad/indices/AnomalyDetectionIndices.java

```diff
@@ -919,10 +919,7 @@ private void updateMappingIfNecessary(GroupedActionListener<Void> delegateListen
                     adminClient
                         .indices()
                         .putMapping(
-                            new PutMappingRequest()
-                                .indices(adIndex.getIndexName())
-                                .type(CommonName.MAPPING_TYPE)
```

Comment:
if all uses of `CommonName.Mapping_TYPE` were removed we can maybe delete that constant and comment referring elastic above it in `CommonName` class

Reply:
Good call out.would do that in separate PR.This is already many files

Reply:
Sounds good